corehq/apps/registry/helper.py

```diff
@@ -26,3 +27,16 @@ def visible_domains(self):
     def participating_domains(self):
         self.registry.check_ownership(self.current_domain)
         return self.registry.get_participating_domains()
+
+    def get_case(self, case_id, case_type):
+        if case_type not in self.registry.wrapped_schema.case_types:
+            raise RegistryAccessException(f"'{case_type}' not available in registry")
+
+        from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+        case = CaseAccessorSQL.get_case(case_id)
```

Comment:
What happens if no case for `case_id` exists?

Reply:
it will raise a `CaseNotFound` exception